launch_ros_sandbox/actions/load_docker_nodes.py

```diff
@@ -43,7 +43,7 @@
 def _containerized_cmd(entrypoint: str, package: str, executable: str) -> List[str]:
     """Prepare the command for executing within the Docker container."""
     # Use ros2 CLI command to find the executable
-    return [entrypoint, 'ros2', 'run', package, executable]
+    return entrypoint.split() + ['ros2', 'run', package, executable]
```

Comment:
Using `shlex.split` would be preferable: https://docs.python.org/3/library/shlex.html#shlex.split

Reply:
Good recommendation